WindDirectionControl/WindDirectionControl.js

```diff
@@ -0,0 +1,120 @@
+/**
+ * Agate styled WindDirectionControl component and behavior.
+ *
+ * @example
+ * <WindDirectionControl />
+ *
+ * @module agate/WindDirectionControl
+ * @exports WindDirectionControl
+ * @exports WindDirectionControlBase
+ * @exports WindDirectionControlDecorator
+ */
+
+import kind from '@enact/core/kind';
+import Changeable from '@enact/ui/Changeable';
+import PropTypes from 'prop-types';
+import compose from 'ramda/src/compose';
+import React from 'react';
+
+import ArcPicker from '../ArcPicker';
+import Icon from '../Icon';
+import Skinnable from '../Skinnable';
+
+import css from './WindDirectionControl.module.less';
+
+/**
+ * An Agate component for displaying Wind Direction Control {@link agate/WindDirectionControl}.
+ *
+ * @class WindDirectionControl
+ * @memberof agate/WindDirectionControl
```

Comment:
Please add @extends JSDoc if using other components in render().
```suggestion
 * @extends agate/ArcPicker.ArcPicker
```